server/routes/testutils/appSetup.ts

```diff
@@ -34,6 +35,12 @@ export const user: Express.User = {
 
 const bookerReference = TestData.bookerReference().value
 
+export type FlashErrors = FieldValidationError[]
```

Comment:
Moving these type definitions to `appSetup.ts` as they're currently only used in tests.